tests/test_ote_api.py

```diff
@@ -437,6 +439,25 @@ def train_and_eval(self, template_dir):
                             f'({validation_performance.score.value} vs {export_performance.score.value}) was '
                             f'larger than the tolerance of {perf_delta_tolerance}')
 
+            print('Run POT optimization.')
+            optimized_model = OptimizedModel(
+                NullProject(),
+                NullModelStorage(),
+                dataset,
+                detection_environment.get_model_configuration(),
+                ModelOptimizationType.POT,
+                optimization_methods=[],
+                optimization_level={},
+                precision=[ModelPrecision.FP16],
+                target_device=TargetDevice.CPU,
+                performance_improvement={},
+                model_size_reduction=1.,
+                model_status=ModelStatus.NOT_READY)
+            ov_task.optimize(OptimizationType.POT, dataset, optimized_model, OptimizationParameters())
+
+            pot_performance = self.eval(ov_task, optimized_model, val_dataset)
+            print(f'Performance of optimized model: {pot_performance.score.value:.4f}')
```

Comment:
So this test only verifies, that optimization and evaluation after it finish without any errors raised, but does it make sense to also check that result is valid?

Reply:
Added